modules/ServerAPI/src/client/Protocol.coffee

```diff
@@ -102,7 +102,17 @@ class Protocol extends BaseEntity
 			maxY.set numericValue: 120.0
 
 		maxY
+			
+	getStrictEndpointMatching: ->
+		strictEndpointMatching = @.get('lsStates').getOrCreateValueByTypeAndKind "metadata", "protocol metadata", "codeValue", "strict endpoint matching"
+		if strictEndpointMatching.get('codeValue') is undefined or strictEndpointMatching.get('codeValue') is ""
+			strictEndpointMatching.set codeValue: "false"
+			strictEndpointMatching.set codeType: "boolean"
+			strictEndpointMatching.set codeKind: "boolean"
+			strictEndpointMatching.set codeOrigin: "ACAS DDICT"
 
+		strictEndpointMatching
+		
```

Comment:
Function for setting up and editing string endpoint matching lsState in the Protocol data